docs/source/quantization.rst

```diff
@@ -80,7 +80,10 @@ The corresponding implementation is chosen automatically based on the PyTorch bu
 Quantization API Summary
 ---------------------------------------
 
-There are three types of quantization supported in PyTorch:
+Eager Mode Quantization
```

Comment:
Should we add a quick section for context on Eager vs FX Graph (why both, what are the tradeoffs, etc)?  Doesn't have to be this PR.